src/extension.ts

```diff
@@ -872,6 +872,7 @@ class ExtensionManager implements vscode.Disposable {
 
   async hideLaunchCommand(shouldHide: boolean = true) {
     // Don't hide command selectLaunchTarget here since the target can still be useful, one example is ${command:cmake.launchTargetPath} in launch.json
+    this._statusBar.hideLaunchButton(shouldHide);
```

Comment:
>  I don't remember why this wasn't added previously.

there was previously no status bar button for that.
i added that line because it looks like the same as with debug.

Reply:
@bobbrow Yes this is a newly added button. From our perspective this looks good to me.